test/optim/test_zeroptimizer.py

```diff
@@ -0,0 +1,469 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
```

Comment:
Let's sync expected license content with MSFT folks on Slack

Reply:
hmm, there's no code from DeepSpeed in there

Reply:
We borrowed the name `ZeRO` :)

Reply:
ah yes, fair enough ! :D